superset-frontend/plugins/plugin-chart-table/src/TableChart.tsx

```diff
@@ -438,7 +440,7 @@ export default function TableChart<D extends DataRecord = DataRecord>(
         (config.showCellBars === undefined
           ? showCellBars
           : config.showCellBars) &&
-        (isMetric || isRawRecords) &&
+        (isMetric || isRawRecords || key[0] === '%') &&
```

Comment:
`DataColumnMeta` has a `isPercentMetric` property.
```suggestion
        (isMetric || isRawRecords || isPercentMetric) &&
```